<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDeviceFileDialog</class>
 <widget class="QDialog" name="CDeviceFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="1">
    <widget class="QToolButton" name="nextToolButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="canary-img.qrc">
       <normaloff>:/img/ic_forward</normaloff>
       <normalon>:/img/ic_forward_white</normalon>:/img/ic_forward</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QToolButton" name="prevToolButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="canary-img.qrc">
       <normaloff>:/img/ic_back</normaloff>
       <normalon>:/img/ic_back_white</normalon>:/img/ic_back</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="pathLineEdit"/>
   </item>
   <item row="3" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTableWidget" name="fileTableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>name: </string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLineEdit" name="nameLineEdit">
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="canary-img.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CDeviceFileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CDeviceFileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
